### PR TITLE
Backport from PR 15160 (cosmic rate tool)  to CMSSW80X 

### PR DESCRIPTION
**Pull request by skeshri**
Backport for the PR 15160 (Cosmic rate tool).

latest presetation(will present next week) : https://cernbox.cern.ch/index.php/s/uNpVR3EYnz7EFcb
presented in Tk alignment meeting on 22 June : https://indico.cern.ch/event/536850/contributions/2215484/attachments/1296737/1933823/TkAlmeeting_22_June_2016.pdf

This tool is useful to calculate cosmic event and track rate. For more details one can browse through the twiki : https://twiki.cern.ch/twiki/bin/viewauth/CMS/TkAlCosmicsRateMonitoring

### CODE REVIEW

File: Alignment/TrackerAlignment/plugins/BuildFile.xml [hunk 1]
```diff
@@ -35,3 +35,38 @@
...
+  <use name="Alignment/CommonAlignment"/>
+  <use name="CondFormats/AlignmentRecord"/>
+  <use name="CondCore/DBOutputService"/>
+  <use name="CondFormats/GeometryObjects"/>
+  <use name="CondFormats/Alignment"/>
+  <use name="DataFormats/TrackerCommon" />
+  <use name="Geometry/Records"/>
+  <use name="Geometry/TrackerGeometryBuilder"/>
+  <use name="CLHEP"/>
+  <flags EDM_PLUGIN="1"/>
```
Comment (ghellwig) [inline Alignment/TrackerAlignment/plugins/BuildFile.xml]: @skeshri this won't compile because this tool is only present in 81X

Comment (ghellwig) [inline Alignment/TrackerAlignment/plugins/BuildFile.xml]: I guess you picked the wrong part in the conflict when rebasing

Comment (skeshri) [inline Alignment/TrackerAlignment/plugins/BuildFile.xml]: I see, Sorry I am updating this!

**Comment by mmusich** [comment]
please test

**Comment by mmusich** [comment]
please test

**Comment by ghellwig** [comment]
+1

**Comment by mmusich** [comment]
@skeshri please change the title to something more meaningful

**Comment by davidlange6** [comment]
+1

**Pull request merged**